### ISSUE TITLE
[Error]   Build failed The command '/bin/sh -c apk update && apk add jq' returned a non-zero code: 1

### ISSUE THREAD

**Issue by lyly0906** [balena-io/balena-cli#1389]
balena login is ok!
![image](https://user-images.githubusercontent.com/26811002/62696084-43b6bf00-ba0a-11e9-9ad0-447682a4e7aa.png)

balena devices is ok!
![image](https://user-images.githubusercontent.com/26811002/62696015-22ee6980-ba0a-11e9-9103-e3bb1eafdd19.png)

but ,deploy is wrong! how can i resovle this problem?
![image](https://user-images.githubusercontent.com/26811002/62696209-7e205c00-ba0a-11e9-9599-076717210cd2.png)

root@ubuntu:/opt/balena-sense-hat-python-starter# balena build --deviceType raspberrypi3 --arch armv7hf --emulated
[Info]    Creating default composition with source: /opt/balena-sense-hat-python-starter
[Info]    Building for armv7hf/raspberrypi3
[Info]    Emulation is enabled
[Build]   Built 1 service in 0:03
[Build]   main Preparing...
[Error]   Build failed
The command '/bin/sh -c apk update && apk add jq' returned a non-zero code: 1

Additional information may be available by setting a DEBUG=1 environment
variable: "set DEBUG=1" on a Windows command prompt, or "export DEBUG=1"
on Linux or macOS.

If you need help, don't hesitate in contacting our support forums at
https://forums.balena.io

For bug reports or feature requests, have a look at the GitHub issues or
create a new one at: https://github.com/balena-io/balena-cli/issues/

**Comment by pdcastro** [comment]
@lyly0906, thanks for getting in touch. The key error message seems to be:
```
The command '/bin/sh -c apk update && apk add jq' returned a non-zero code: 1
```
It means that the following command failed to execute:
```
apk update && apk add jq
```
I assume / expect that the command above was executed (attempted) during the image build, perhaps as part of a RUN instruction of a Dockerfile, even though I don't see that command in the Dockerfile of the `balena-sense-hat-python-starter` project:
https://github.com/balena-io-playground/balena-sense-hat-python-starter/blob/master/Dockerfile

In any case, this sounds like a general support / troubleshooting issue with balena deployments or the `balena-sense-hat-python-starter` project, rather a `balena-cli` bug, so for further support I would suggest that a thread was created in the balenaCloud forums:
https://forums.balena.io/c/balena-cloud

balena's support agents monitor the forums (in addition to the wider community), so you should quickly get further help over there.

**Comment by lyly0906** [comment]
Thanks！ 
The link is  https://www.youtube.com/watch?v=3eVwLxfh3k8 .(Setting up openBalena on a VPS (stream archive))  I watched this video in youtube.com . My computer's system is ubuntu 18.04 and balena-cli is 11.7.8.  
![image](https://user-images.githubusercontent.com/26811002/62754260-ab6c1900-baa1-11e9-85bd-bfc0a6bf1aa7.png)

why， when  i deploy the same project ， wrong is “apk update && apk add jq” ？

**Comment by pdcastro** [comment]
> why, when i deploy the same project, wrong is “apk update && apk add jq” ？

@lyly0906, this sounds like a general support / troubleshooting issue rather than a balena-cli bug, so I suggest that you create a thread in the openBalena forum:
> https://forums.balena.io/c/open-balena

balena's support agents do not monitor this balena-cli github repo, but they monitor the forums, so you should quickly get further help over there. Thanks.

**Comment by xginn8** [comment]
this issue looks related to https://github.com/balena-io/balena-cli/issues/1355, perhaps better to debug there? (cc @pdcastro)

**Comment by pdcastro** [comment]
@lyly0906, this issue was fixed in CLI [v11.9.0](https://github.com/balena-io/balena-cli/releases). Sorry for initially misunderstanding that it was indeed a bug. Thanks to you and to @xginn8 for reporting it.
